tribe-ext-eventbrite-addl-opts/index.php

```diff
@@ -230,35 +226,47 @@ public function eb_currency_for_update_create( $args, $mode, $eventbrite_id, $po
 	/**
 	 * List of locales supported by Eventbrite
 	 * This list is hand-crafted from the Region Selector in the global footer of Eventbrite.com pages. Screenshot at https://cl.ly/412T23182w1Q
+	 *
 	 * @access private
+	 *
 	 * @return array
 	 */
 	private function eb_locale_country_name_array() {
 		$array = array(
-			'es_AR'		=> __( 'Argentina', 'tribe-extension' ),
-			'en_AU'		=> __( 'Australia', 'tribe-extension' ),
-			'pt_BR'		=> __( 'Brazil', 'tribe-extension' ),
-			'en_CA'		=> __( 'Canada (EN)', 'tribe-extension' ),
-			'fr_CA'		=> __( 'Canada (FR)', 'tribe-extension' ),
-			'de_DE'		=> __( 'Germany', 'tribe-extension' ),
-			'es_ES'		=> __( 'Spain', 'tribe-extension' ),
-			'fr_FR'		=> __( 'France', 'tribe-extension' ),
-			'en_HK'		=> __( 'Hong Kong', 'tribe-extension' ),
-			'en_IE'		=> __( 'Ireland', 'tribe-extension' ),
-			'it_IT'		=> __( 'Italy', 'tribe-extension' ),
-			'nl_NL'		=> __( 'Netherlands', 'tribe-extension' ),
-			'en_NZ'		=> __( 'New Zealand', 'tribe-extension' ),
-			'pt_PT'		=> __( 'Portugal', 'tribe-extension' ),
-			'en_SG'		=> __( 'Singapore', 'tribe-extension' ),
-			'en_GB'		=> __( 'United Kingdom', 'tribe-extension' ),
-			'en_US'		=> __( 'United States', 'tribe-extension' ),
+			'es_AR' => __( 'Argentina', 'tribe-extension' ),
+			'en_AU' => __( 'Australia', 'tribe-extension' ),
+			'pt_BR' => __( 'Brazil', 'tribe-extension' ),
+			'en_CA' => __( 'Canada (EN)', 'tribe-extension' ),
+			'fr_CA' => __( 'Canada (FR)', 'tribe-extension' ),
+			'de_DE' => __( 'Germany', 'tribe-extension' ),
+			'es_ES' => __( 'Spain', 'tribe-extension' ),
+			'fr_FR' => __( 'France', 'tribe-extension' ),
+			'en_HK' => __( 'Hong Kong', 'tribe-extension' ),
+			'en_IE' => __( 'Ireland', 'tribe-extension' ),
+			'it_IT' => __( 'Italy', 'tribe-extension' ),
+			'nl_NL' => __( 'Netherlands', 'tribe-extension' ),
+			'en_NZ' => __( 'New Zealand', 'tribe-extension' ),
+			'pt_PT' => __( 'Portugal', 'tribe-extension' ),
+			'en_SG' => __( 'Singapore', 'tribe-extension' ),
+			'en_GB' => __( 'United Kingdom', 'tribe-extension' ),
+			'en_US' => __( 'United States', 'tribe-extension' ),
 		);
 
 		$array = $this->append_array_keys_to_values( $array );
 
 		return $array;
 	}
 
+	/**
+	 * Joins the array keys with its values
+	 * Used for display purposes in the settings/choices
```

Comment:
A few of your docblocks do not include the space between the summary and the description. This will prevent them from being parsed properly. Checkout this [handy guide](https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/php/#docblock-formatting) for more info.

Also, many are missing the trailing period on each sentence -- a super small thing but I just though I'd mention it in case you hadn't noticed.